src/utils.js

```diff
@@ -12,6 +12,43 @@ import { PACKAGE_TYPES, LOCAL_PROTOCOLS } from 'const';
 /* global nodeRequire, localesRoot */
 
 const SOURCE_MAP_RE = /\/\/[#@]\s(source(?:Mapping)?URL)=\s*(\S+)/;
+// Firefox's version format is laxer than Chrome's, it accepts:
+// https://developer.mozilla.org/en-US/docs/Toolkit_version_format
+// We choose a slightly restricted version of that format (but still more
+// permissive than Chrome) to allow Beta addons, per:
+// https://developer.mozilla.org/en-US/Add-ons/AMO/Policy/Maintenance
```

Comment:
This one also doesn't exist anymore, I think we may have moved this content to extensionworkshop, but we would need to look into what the old MDN page used to contain to be sure of which one of the extensionworkshop pages may have replaced it

I'd guess it may be https://extensionworkshop.com/documentation/publish/distribute-pre-release-versions/?

Reply:
I changed the link to EW, at least the page loads.